package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "Mozilla-Kerala",
+  "description": "Mozilla Kerala Website",
+  "version": "0.0.1",
+  "license": "SEE LICENSE IN LICENSE",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/Mozilla-Kerala/home-front.git"
+  },
+  "scripts": {
+    "ng": "ng",
+    "start": "ng serve",
+    "build": "ng build",
+    "lint": "ng lint ng-universal-demo",
+    "build:client-and-server-bundles": "ng build --prod && ng run ng-universal-demo:server:production",
+    "build:prerender": "npm run build:client-and-server-bundles && npm run compile:server && npm run generate:prerender",
+    "build:ssr": "npm run build:client-and-server-bundles && npm run compile:server",
+    "compile:server": "tsc -p server.tsconfig.json",
+    "generate:prerender": "cd dist && node prerender",
+    "serve:prerender": "cd dist/browser && http-server",
+    "serve:ssr": "node dist/server"
+  },
+  "dependencies": {
```

Comment:
can we create a new structure with angular 7 dependencies this looks for angular 6

Reply:
Will update with angular 7